cdap-cli/src/main/java/co/cask/cdap/cli/command/stream/view/DescribeStreamViewCommand.java

```diff
@@ -59,11 +59,12 @@ public void perform(Arguments arguments, PrintStream output) throws Exception {
     ViewDetail detail = client.get(Id.Stream.View.from(streamId, arguments.get(ArgumentName.VIEW.toString())));
 
     Table table = Table.builder()
-      .setHeader("id", "format", "schema", "settings")
+      .setHeader("id", "format", "table", "schema", "settings")
       .setRows(Collections.singletonList(detail), new RowMaker<ViewDetail>() {
         @Override
         public List<?> makeRow(ViewDetail object) {
           return Lists.newArrayList(object.getId(), object.getFormat().getName(),
+                                    object.getTableName(),
```

Comment:
Always the actual table name. Table name is only nullable in client code or `ViewAdmin.createOrUpdate`. A persisted `ViewSpecification` always has an actual table name.